.github/labeler.yml

```diff
@@ -0,0 +1,53 @@
+# Add 'python' label to any .py file changes
```

Comment:
Noticed this about the same YAML file in `public-api-service`, but there's no `go` label?

Reply:
yeah I just slurped this from our repo 😹